### PR TITLE
Playable version messages

### PR DESCRIPTION
**Pull request by FlorisXll**
Some tests could still be added and I'm not sure if the Travis build will fail or not.

Also the client must be merged together with this one. https://github.com/jessetilro/pixelperfect-client/pull/11

### CODE REVIEW

**Comment by dmalarev** [comment]
looks good

File: src/test/java/nl/tudelft/pixelperfect/client/ServerListenerTest.java [hunk 1]
```diff
@@ -52,25 +52,25 @@ public void testGetGame() {
...
+//   */
+//  @SuppressFBWarnings("RV_RETURN_VALUE_IGNORED_NO_SIDE_EFFECT")
+//  @Test
+//  public void testMessageReceived() {
+//    // Fixtures
+//    HostedConnection mockedSource = mock(HostedConnection.class);
+//    EventCompletedMessage mockedMessage = mock(EventCompletedMessage.class);
+//
+//    // Stubbing
+//    when(mockedMessage.getCompletedEvent()).thenReturn(0);
```
Comment (wouterz) [inline src/test/java/nl/tudelft/pixelperfect/client/ServerListenerTest.java]: should not contain commented code

**Comment by wouterz** [comment]
rest looks good

File: src/test/java/nl/tudelft/pixelperfect/client/ServerListenerTest.java [hunk 2]
```diff
@@ -112,7 +112,7 @@ public void testCompletedEventMessageReceived() {
     object.messageReceived(mockedSource, mockedMessage);
 
     // Verification
```
Comment (dalderliesten) [inline src/test/java/nl/tudelft/pixelperfect/client/ServerListenerTest.java]: Inline comments are nice, but they should not be used for obvious things.

**Comment by dalderliesten** [comment]
Beyond Wouter and my comments, looks good.

File: src/main/java/nl/tudelft/pixelperfect/client/EventCompletedMessage.java [hunk 3]
```diff
@@ -25,7 +26,8 @@ public EventCompletedMessage() {
    * 
    * @param completed , a completed Event.
    */
-  public EventCompletedMessage(String completed) {
+  public EventCompletedMessage(String label, int completed) {
```
Comment (dmalarev) [inline src/main/java/nl/tudelft/pixelperfect/client/EventCompletedMessage.java]: There's a param tag missing for label.

**Comment by dmalarev** [comment]
I think the last issue is that getLabel() is not tested in EventMessage

**Comment by FlorisXll** [comment]
Correct, I will do that now.

**Comment by FlorisXll** [comment]
Correction, I did test this already, you look at the wrong test class (should be EventCompletedMessageTest).

**Comment by dmalarev** [comment]
It's there, but it doesnt run it because you forgot the test tag before the case

File: src/test/java/nl/tudelft/pixelperfect/client/EventCompletedMessageTest.java [hunk 4]
```diff
@@ -31,14 +31,21 @@ public void init() {
...
   @Test
   public void testGetCompletedEvent() {
-    assertThat(object.getCompletedEvent(), is("Hello World!"));
+    assertThat(object.getCompletedEvent(), is(0));
+  }
+  
+  /**
+   * Testing the getLabel method.
+   */
+  public void testGetLabel() {
```
Comment (dmalarev) [inline src/test/java/nl/tudelft/pixelperfect/client/EventCompletedMessageTest.java]: Missing Test tag

**Comment by dmalarev** [comment]
LGTM

**Comment by wouterz** [comment]
LGTM

**Pull request merged**